Mmkbfa

MuffinManKen opened the pull request:
Addresses:
Issue #44 
Issue #48 Garrison Resources can be purchased with alt-click again

MuffinManKen committed:
Fixes issue #44 which still happens on occasion

MuffinManKen committed:
Issue #48: Garrison Resources can be purchased with alt-click again

ckaotik commented:
For the garrison resources, could you please use `GetCurrencyInfo(824)` to get the name, for languages other than English? :)

MuffinManKen committed:
Changed look up to use currency ID rather than an English name (thanks ckaotic)
Uses a key lookup instead of a string compare and easily expands to support any other currencies that this would be needed for

Myrroddin commented:
Line 35 of ui-scrollbar.lua in the externals folder must be changed to
`PlaySoundKit(SOUNDKIT.UChatScrollButton)`

spiralofhope commented:
@MuffinManKen - In case you're still interested in the updates to this addon, https://github.com/TekNoLogic/GnomishVendorShrinker/pull/55 has additional patches.
